repotracker/github_poller.go

```diff
@@ -58,8 +57,7 @@ func githubCommitToRevision(
 
 // GetRemoteConfig fetches the contents of a remote github repository's
 // configuration data as at a given revision
-func (gRepoPoller *GithubRepositoryPoller) GetRemoteConfig(
-	projectFileRevision string) (projectConfig *model.Project, err error) {
+func (gRepoPoller *GithubRepositoryPoller) GetRemoteConfig(projectFileRevision string) (projectConfig *model.Project, err error) {
```

Comment:
Could we declare the return variables inside the function rather than in the signature?